src/gleam/float.gleam

```diff
@@ -384,3 +388,15 @@ if javascript {
   external fn do_random_uniform() -> Float =
     "../gleam_stdlib.mjs" "random_uniform"
 }
+
+pub fn add(a: Float, b: Float) -> Float {
+  a +. b
+}
+
+pub fn multiply(a: Float, b: Float) -> Float {
+  a *. b
+}
+
+pub fn subtract(minuend a: Float, subtrahend b: Float) -> Float {
+  a -. b
+}
```

Comment:
Remove these please, we avoid having multiple ways of doing the same thing in Gleam so we don't want to have functions that do the same as the operators.

Reply:
OK, I will repackage those eventually because they are nice for piping calculus,